pages/index.tsx

```diff
@@ -1,15 +1,42 @@
 import Head from 'next/head';
-import {Container, Row, Col, Jumbotron, Image, Carousel} from 'react-bootstrap';
-import {BiRecycle} from 'react-icons/bi';
+import { Container, Row, Col, Jumbotron, Image, Carousel, Accordion, Card, Button, AccordionContext, useAccordionToggle } from 'react-bootstrap';
+import { BiRecycle } from 'react-icons/bi';
 
 import etcStyles from '../styles/etc.module.css';
 
 import Navigation from '../components/navbar';
 import Footer from '../components/footer';
+import { useContext } from 'react';
+
+interface Toggle{
```

Comment:
nit: spacing

interface Toggle {

Reply:
same below for HomeContent.

btw, I'm surprised typescript doesn't format this automatically. maybe there's a VSCode plugin that can do this?